benchmark/events/events-newListener.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
```

Comment:
I don't think this file shall be included in this PR. Please try to remove this file :p

Reply:
I am sorry, I confuse the two pr, I will fix it. Thanks!